dvc/command/pkg.py

```diff
@@ -4,27 +4,92 @@
 import logging
 
 from dvc.exceptions import DvcException
-from dvc.command.base import CmdBase, fix_subparsers, append_doc_link
+
+from .config import CmdConfig
+from .base import CmdBase, fix_subparsers, append_doc_link
+
+from dvc.pkg.config import pkg_add, pkg_remove, pkg_modify, pkg_list
 
 
 logger = logging.getLogger(__name__)
 
 
+class CmdPkgAdd(CmdConfig):
+    def run(self):
+        pkg_add(
+            self.config, self.args.name, self.args.url, level=self.args.level
+        )
+        return 0
+
+
+class CmdPkgRemove(CmdConfig):
+    def run(self):
+        pkg_remove(self.config, self.args.name, level=self.args.level)
+        return 0
+
+
+class CmdPkgModify(CmdConfig):
+    def run(self):
+        pkg_modify(
+            self.config,
+            self.args.name,
+            self.args.option,
+            self.args.value,
+            level=self.args.level,
+        )
+        return 0
+
+
+class CmdPkgList(CmdConfig):
+    def run(self):
+        pkgs = pkg_list(self.config, level=self.args.level)
+        for name, url in pkgs.items():
+            logger.info("{}\t{}".format(name, url))
+        return 0
+
+
 class CmdPkgInstall(CmdBase):
+    def run(self):
+        ret = 0
+        for target in self.args.targets:
+            try:
+                self.repo.pkg.install(name=target)
+            except DvcException:
```

Comment:
why do we catch this generic exception in some `runs` and don't catch in others? Would it be easier to always catch it in CmdBase or in some decorator at lease and don't duplicate these `except DvcException`?

Reply:
Because error message depends on where it is caught. Pkg add/remove/midify are dealing with configs, so they are raising ConfigError, which is dealt with in main.py.

Reply:
But if the only difference is the message it should be easy to wrap it (into decorator?) and remove a lot of duplication.

Reply:
Not sure I see a clear way to do that nicely for all commands, especially for the ones that process multiple targets. Or are you only bothered by Install/Uninstall? To me, this is a case where copypasting for the sake of simplicity is ok.

Reply:
Don't like the duplication :) But agree that it's fine for now. In the future I would try something like:

```
@dvc_exception("Failed to install the package")
def install(...)
    blah
    blah
```

Yes, it not customizable, but less duplication, simpler to read, manage, etc. 

If we need some customization @dvc_exception can parse and append function arguments.